platform/MCU/newhal-mcu/inc/hw_ticks.h

```diff
@@ -48,9 +48,9 @@ void System1UsTick(void);
 /**
  * Fetch the current milliseconds count.
  * @return the number of milliseconds since the device was powered on or woken up from
- * sleep. Automatically wraps around when above UINT_MAX;
+ * sleep.
  */
-system_tick_t GetSystem1MsTick();
+uint64_t GetSystem1MsTick();
```

Comment:
This is an open question - do we break compatibility of these low-level functions? My instinct is to say no and move the 64-bit version to `GetSystem1MsTick64` or similar, but happy to hear arguments to the contrary.

Reply:
👍